guides/release/routing/loading-and-error-substates.md

```diff
@@ -206,7 +204,7 @@ called with the `error` as the model. See example below:
 ```javascript
 setupController(controller, error) {
   console.log(error.message);
-  this._super(...arguments);
+  super(...arguments);
```

Comment:
This should be `super.setupController(...arguments)` (`super()` only works in constructors)

Reply:
done